src/views/shared/ipv-template.html

```diff
@@ -4,49 +4,12 @@
 {% from "govuk/components/phase-banner/macro.njk"           import govukPhaseBanner %}
 {% from "govuk/components/warning-text/macro.njk"           import govukWarningText %}
 {% from "govuk/components/details/macro.njk"                import govukDetails %}
+{% from "govuk/components/cookie-banner/macro.njk"          import govukCookieBanner %}
 
 {% set hmpoPageTitle = "pages."+hmpoPageKey+".title" %}
 {% set hmpoPageTitleServiceNameRequired = "pages."+hmpoPageKey+".serviceNameRequired" %}
 {% set hmpoPageHtmlContent = "pages."+hmpoPageKey+".htmlContent" %}
 
-
-{% block head %}
-{# super copies over the code for this block from HMPO template #}
-{{ super() }}
-<script>
```

Comment:
This part of the commit is an earlier version of the GA code that we added to check the tracking code was working. It's all handled in application.js now and we can’t use the `<noscript>` code because it opts people into tracking without their consent. The nunjucks block removes the cookie banner from the source so we need to go back to the default behaviour there.